single-node/user-data

```diff
@@ -83,7 +83,7 @@ function init_templates {
         mkdir -p $(dirname $TEMPLATE)
         cat << EOF > $TEMPLATE
 [Service]
-ExecStartPre=/usr/bin/mkdir -p /etc/kubernetes/manifests
```

Comment:
It is a long standing convention - there is no need to change it now.

Can we just go with this:

`/etc/kubernetes/manifests` = static pods
`/etc/kubernetes/addons` = addons which will be created with `kubectl create -f`
`/srv/kubernetes/manifests` is reserved, but fine if unused.
```